scripts/modules/elastic_stack.py

```diff
@@ -587,6 +587,46 @@ def render_tee(self):
         return {self.name(): content}
 
 
+class PackageRegistry(StackService, Service):
+
+    SERVICE_PORT = "8080"
+
+    docker_path = "package-registry"
+    docker_name = "distribution"
+
+    def __init__(self, **options):
+        super(PackageRegistry, self).__init__(**options)
+        self.distribution = options.get("package_registry_distribution", "snapshot")
+        self.apm_package = options.get("apm_package")
+        if not self.at_least_version("7.10"):
+            raise Exception("Package registry only supported for 7.70+")
+        self.environment = {}
+
+    def _content(self):
+        content = dict(
+            image="/".join((self.docker_registry, self.docker_path, self.docker_name)) + ":" + self.distribution,
```

Comment:
can this be:
```suggestion
            image=self.default_image(version_override=self.distribution),
```

Reply:
This won't work because the package-registry images don't follow conventions, so all those `-SNAPSHOT`, `-oss` suffixes are not there.

Implemented the rest, thanks for the review!

Reply:
Thanks for checking